spec/reactors/mempool/reactor.md

```diff
@@ -13,6 +13,11 @@ for details.
 Sending incorrectly encoded data or data exceeding `maxMsgSize` will result
 in stopping the peer.
 
+`maxMsgSize` equals `MaxTxBytes` (1MB) * `TxBatchSize` (100) + 4 (proto
+overhead). `MaxTxBytes` and `TxBatchSize` are mempool config parameters ->
+defined locally. The reactor sends transactions to the connected peers in
+batches. The maximum size of one batch is `TxBatchSize`.
```

Comment:
Do peers get told your MaxTxBytes and TxBatchSize in the handshake?

Reply:
Not yet -> https://github.com/tendermint/tendermint/issues/5332